src/components/ui/Map.styled.js

```diff
@@ -1,8 +1,13 @@
 import styled from 'styled-components';
 
-export const StyledMap = styled.div`
-	height: 30rem;
-	max-width: 40rem;
+const StyledMap = styled.section`
+	margin: auto;
+	border: 2px solid var(--main-bg-color-full);
```

Comment:
What's the difference between `--main-bg-color-full` and `--main-bg-color`? Usually I'd probably want my variables to be fairly verbose - meaning: I want to be able to identify their purpose as quickly as possible. :) I'm not sure the appendix -full is very verbose. What do you think?